.github/workflows/test-terra-jupyter-aou.yml

```diff
@@ -61,56 +64,56 @@ jobs:
         gcloud auth configure-docker
         ./build_smoke_test_image.sh terra-jupyter-aou
 
-    - name: Run Python code specific to notebooks with nbconvert
-      # Run all notebooks from start to finish, regardles of error, so that we can capture the
-      # result as a workflow artifact.
-      # See also https://github.com/marketplace/actions/run-notebook if a more complicated
-      # workflow for notebooks is needed in the future.
-      run: |
-        chmod a+w -R $GITHUB_WORKSPACE
-        docker run \
-          --env GOOGLE_PROJECT \
-          --volume "${{ env.GOOGLE_APPLICATION_CREDENTIALS }}:/tmp/credentials.json:ro" \
-          --env GOOGLE_APPLICATION_CREDENTIALS="/tmp/credentials.json" \
-          --volume $GITHUB_WORKSPACE:/tests \
-          --workdir=/tests \
-          --entrypoint="" \
-          terra-jupyter-aou:smoke-test \
-          /bin/bash -c 'for nb in {terra-jupyter-python/tests,terra-jupyter-aou/tests}/*ipynb ; do jupyter nbconvert --to html --ExecutePreprocessor.allow_errors=True --execute "${nb}" ; done'
-
-    - name: Upload workflow artifacts
-      uses: actions/upload-artifact@v2
-      with:
-        name: notebook-execution-results
-        path: |
-          terra-jupyter-python/tests/*.html
-          terra-jupyter-aou/tests/*.html
-        retention-days: 30
+    # - name: Run Python code specific to notebooks with nbconvert
```

Comment:
Shall we make a ticket to re-enable aou smoke tests?

Reply:
I was hoping we can resolve it this sprint. If not, I'll create a separate ticket

Reply:
I will try to figure out the problem with the one terra_widgets test.

In general, it's better to disable the single failing test than the whole test suite (but I know you folks are just trying to get this wrapped up).

And it appears that its not an option to just leave `terra-jupyter-aou` with its current docker version? The AoU team will need to remember not to upgrade to the newer image. It would be better if they didn't need to remember this.

Reply:
@deflaux I was hoping we'll figure this out this week...if not, I'll bring more attention to this from AoU team

Reply:
got it, thanks @Qi77Qi !